lib/registry.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+var Report = require('metrics').Report,
+  util = require('util');
+
+var Registry = module.exports = function (trackedMetrics) {
+  Registry.super_.call(this, trackedMetrics);
+  this.trackedTags = {};
+}
+
+util.inherits(Registry, Report);
```

Comment:
I think we can use ES6 class instead for inheritance
https://nodejs.org/docs/latest/api/util.html#util_util_inherits_constructor_superconstructor